pkg/integration/components/test_driver.go

```diff
@@ -84,9 +83,7 @@ func (self *TestDriver) Shell() *Shell {
 // in the current page and failing that, jump to the top of the view and iterate through all of it,
 // looking for the item.
 func (self *TestDriver) navigateToListItem(matcher *matcher) {
-	self.inListContext()
-
-	currentContext := self.gui.CurrentContext().(types.IListContext)
```

Comment:
turns out it doesn't matter if we're in a list item or not. I'll update this function to be called `navigateToLine` in a separate PR